src/applications/disability-benefits/686/components/AuthorizationMessage.jsx

```diff
@@ -0,0 +1,67 @@
+import SystemDownView from '@department-of-veterans-affairs/formation/SystemDownView';
+import React from 'react';
+import appendQuery from 'append-query';
+import PropTypes from 'prop-types';
+
+import manifest from '../../526EZ/manifest.json';
+
+const { rootUrl: increaseRootUrl } = manifest;
+import { profileStatuses } from '../helpers';
+
+const { serverError, notFound } = profileStatuses;
+const nextQuery = { next: window.location.pathname };
+const signInUrl = appendQuery('/', nextQuery);
+const verifyUrl = appendQuery('/verify', nextQuery);
+
+export default class AuthorizationMessage extends React.Component {
```

Comment:
This logic was split out in an effort to prepare to make the authorization logic/messaging reusable for different forms down the line.

Reply:
Thanks @U-DON, I've created an [issue](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13797) to review this content.